src/main/app.ts

```diff
@@ -40,12 +41,15 @@ new CSRFToken().enableFor(app);
 new Nunjucks(developmentMode).enableFor(app);
 new OidcMiddleware().enableFor(app);
 new HealthCheck().enableFor(app);
+new Container().enableFor(app);
 
 logger.info('Environment: ' + env);
 
-setupDev(app,developmentMode);
+setupDev(app, developmentMode);
 setupTest(app);
 
+app.locals.container.cradle.autoSuggestService.loadData(developmentMode || env === 'test');
```

Comment:
I would look into moving the loading of autoSuggestService into its own module. this module could then do something like the following

Module
```
export class AutoSuggest {

  constructor(
   private autoSuggestService: autoSuggestInterface
   private developmentMode: boolean
   ) {}
  
 enableFor(app: express.Express): void {

    app.use((req, res, next) => {
    //res.locals are available throughout the application including nunjucks so no need to pass it through on all methods
     res.locals.jurisdictions = autoSuggestService.getJurisdictions();
     res.locals.caseTypes = autoSuggestService.getCaseTypes();
    })
  }

}
```

then in application.ts

```
new AutoSuggest(app.locals.container.cradle.autoSuggestService, developmentMode || env == 'test').enableFor(app);
```